src/common/runtime/standalone.ts

```diff
@@ -84,7 +84,7 @@ dataCache.setStore({
     if (!response.ok) {
       return Promise.reject(response.statusText);
     }
-    return await response.text();
+    return new Uint8Array(await response.arrayBuffer());
```

Comment:
please prepare a CL to dawn to change test_runner.js to do this too, based on the .bin or .json extension? that way the roll will be able to work without manual intervention

Reply:
https://dawn-review.googlesource.com/c/dawn/+/158061